microcosm_fastapi/database/encryption/store.py

```diff
@@ -79,12 +79,12 @@ def reencrypt_instance(self, instance, encryption_context_key):
         return self.model_class(**values)
 
     async def search_encrypted_ids(self, context_id):
-        query = self.session.query(
-            self.model_class.id,
-        ).where(
-            getattr(self.model_class, self.model_class.__encrypted_identifier__) != None,  # noqa
-            getattr(self.model_class, self.model_class.__encryption_context_key__) == context_id,
-        )
-
-        results = await self.session.execute(query)
+        async with self.session_maker() as session:
```

Comment:
Fixed this as part of the move towards using `self.session_maker` in recent microcosm-fastapi version releases (0.0.3/0.0.4) to create sessions rather than using an AsyncSessionContext as part of `0.0.2`